### PR TITLE
Another Test

### PR DESCRIPTION
**Pull request by tstirrat15**
Here we go

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,3 +33,4 @@ The following is a screenshot of the wireframe, highlighting the website's propo
 - Moe:
 - Lauren:
 - Olivia: (TBD on return to class 11/16)
+- Tanner: demo how branches work on github
```
Comment (L-Buchholz) [inline README.md]: Hey, capitalize Github already!

**Review by L-Buchholz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tstirrat15) [reply README.md]:
```suggestion
- Tanner: demo how branches work on Github
```

**Comment by tstirrat15** [comment]
I made the changes. Can you please re-review?

**Review by L-Buchholz — APPROVED** [review]
Thanks for doing that T!

**Pull request merged**